tests/base.py

```diff
@@ -65,9 +68,7 @@ def setUpClass(cls):
         cls.app = create_app('testing', "")
         cls.app_context = cls.app.app_context()
         cls.app_context.push()
-        db.create_all()
```

Comment:
Don't we need to create tables anmore?

Reply:
I moved it into the `create_app()` factory in case the configuration is `"testing"`:
https://github.com/privacyidea/privacyidea/blob/c0c5f2d5eaab75880d05ba1b1716afbcc41f80c5/privacyidea/app.py#L243
But since i have re-introduced the check for the existence of the table, i will test if this is still necessary.

Reply:
I tested it and it s not necessary anymore, i'll revert the changes.

Reply:
Done in 5c5d6fd79